cli/commands/start.go

```diff
@@ -112,7 +112,7 @@ func (cmd *Start) Run(c *cli.Context) error {
 
 	cmd.out.Verbose.Println("Launching Dashboard...")
 	dash := Dashboard{BaseCommand{machine: cmd.machine, out: cmd.out}}
-	dash.LaunchDashboard(cmd.machine)
+	dash.LaunchDashboard(cmd.machine, true)
```

Comment:
If we update every time we start the VM, why would we want to provide the option to not update in the dashboard command. We should remove that and always update.